personalize.admin.inc

```diff
@@ -413,14 +426,15 @@ function personalize_agent_list($agents = NULL, $allow_edit = TRUE, $allow_alter
       array_splice($build_header, 2, 1);
     }
 
-    $build['#attached']['css'][] = drupal_get_path('module', 'personalize') . '/css/personalize.admin.css';
-    $build['table_' . $status] = array(
+    $build['personalization_group_' . $status]['table'] = array(
       '#theme' => 'table',
       '#header' => $build_header,
       '#rows' => $rows,
       '#attributes' => array('id' => 'personalize'),
     );
   }
+  $build['#attached']['css'][] = drupal_get_path('module', 'personalize') . '/css/personalize.admin.css';
+  $build['#attached']['library'][] = array('personalize', 'admin.campaign.list');
 
```

Comment:
Fine to move it... but I'd go one further and just add it to your new library since its required for the page.